OWDTestToolkit/apps/loop.py

```diff
@@ -271,22 +303,13 @@ def phone_login_manually(self, phone_number_without_prefix):
         allow_button.tap()
 
         try:
-            def _allow_contact_permission():
-                self.marionette.switch_to_frame()
-                self.parent.wait_for_element_displayed(*DOM.GLOBAL.app_permission_btn_yes, timeout=30)
-                allow_permission_button = self.marionette.find_element(*DOM.GLOBAL.app_permission_btn_yes)
-                self.UTILS.element.simulateClick(allow_permission_button)
-
-            _allow_contact_permission()
-
-            self.UTILS.iframe.switchToFrame(*DOM.Loop.mobile_id_frame_locator)
-            time.sleep(2)
-
-            self.UTILS.reporting.logResult('info', "Permissions accepted. Click on verified button!!")
-            self.parent.wait_for_element_displayed(*DOM.Loop.mobile_id_verified_button, timeout=5)
+            self.parent.wait_for_element_displayed(
```

Comment:
I think this piece of code does not work right. The code it was before did, though.